stubs/psycopg2/psycopg2/_psycopg.pyi

```diff
@@ -89,7 +89,7 @@ class _SupportsReadAndReadlineAndWrite(_SupportsReadAndReadline, SupportsWrite[s
 class cursor:
     arraysize: int
     binary_types: Incomplete | None
-    connection: _Connection
+    connection: connection
```

Comment:
I am not sure this is safe in all type checkers, since some may understand this as a forward reference. I'd prefer to keep the alias.

Reply:
No problem! I've open https://github.com/python/typeshed/issues/11113 so we don't forget to validate.